### PR TITLE
zuul.yaml: add debian packaging job

### PR DESCRIPTION
**Pull request by fredericlepied**

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -11,6 +11,7 @@ Architecture: all
 Section: python
 Depends: ${python:Depends},
          ${misc:Depends},
+         python3,
```
Comment (sileht) [inline debian/control]: Why, this application is still in python2

Comment (fredericlepied) [reply debian/control]: there is a script that is in python3 (reported by lintian)...

Comment (sileht) [reply debian/control]: Found it: https://github.com/wazo-platform/wazo-agid/blob/master/bin/change-pass-vm

So we also missed some deps:
```
wazo-auth-client-python3
xivo-lib-python-python3
```

Comment (fredericlepied) [reply debian/control]: added

**Pull request merged**